code/go/0chain.net/chaincore/smartcontract/handler_test.go

```diff
@@ -0,0 +1,426 @@
+package smartcontract_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"reflect"
+	"testing"
+
+	"github.com/rcrowley/go-metrics"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	chstate "0chain.net/chaincore/chain/state"
+	"0chain.net/chaincore/smartcontract"
```

Comment:
This is maybe the only use case where it makes sense to use dot imports. `. "0chain.net/chaincore/smartcontract"`. I think using a dot import here helps to make it clear what methods you are testing.